src/cvode/cvode_io.c

```diff
@@ -86,6 +86,29 @@ int CVodeSetUserData(void* cvode_mem, void* user_data)
   return (CV_SUCCESS);
 }
 
+/*
+ * CVodeSetOwnUserData
+ *
+ * Specifies whether the integrator owns the user data pointer.
+ */
+
+int CVodeSetOwnUserData(void* cvode_mem, sunbooleantype own_user_data)
```

Comment:
Prefix internal function with `cv`
```suggestion
int cvSetOwnUserData(void* cvode_mem, sunbooleantype own_user_data)
```